docs/ide-usage.md

```diff
@@ -53,6 +53,14 @@ The following plugins are needed:
      * ORM Enhancement: `App Engine → ORM` → clear all the entries, and add the following entry: `src/main/java/teammates/storage/entity/*.java`.
      * Validation exclusion: `App Engine → Validation` → add two entries: `src/test/java` and `src/client/java`.
      * WAR directory: `Web Application` → tick both `This project has a WAR directory` and `Launch and deploy from this directory`, and enter `src/main/webapp` as `WAR directory`.
+   * JDK: `Java Build Path → Libraries` → ensure that the system library used is JDK 7.
+
+       ![setupguide-7.png](images/setupguide-7.png)
+
+   * Compiler compliance: `Java Compiler` → tick `"Use compliance from execution environment 'JavaSE-1.7' on the 'Java Build Path'" `.
```

Comment:
Remove the double quotes (`"`). Should just be: tick `Use compliance ... Build Path'`.